src/python/nimbusml.pyproj

```diff
@@ -1199,25 +1231,25 @@
     <Content Include="nimbusml\datasets\baseline\featurizeImage_resnet18_output.csv" />
     <Content Include="nimbusml\datasets\baseline\featurizeImage_resnet50_output.csv" />
     <Content Include="nimbusml\datasets\baseline\magic.nn" />
-    <Content Include="nimbusml\datasets\_data\gplv2\airquality.csv" />
-    <Content Include="nimbusml\datasets\_data\gplv2\COPYING" />
-    <Content Include="nimbusml\datasets\_data\gplv2\infert.csv" />
-    <Content Include="nimbusml\datasets\_data\test-msltr.sample.csv" />
-    <Content Include="nimbusml\datasets\_data\test-ticketchoice.csv" />
-    <Content Include="nimbusml\datasets\_data\test-twitter.gen-sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\test.wikipedia.sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\test_fs.csv" />
-    <Content Include="nimbusml\datasets\_data\timeseries.csv" />
-    <Content Include="nimbusml\datasets\_data\topics.csv" />
-    <Content Include="nimbusml\datasets\_data\train-250.wikipedia.sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\train-500.uciadult.sample.csv" />
-    <Content Include="nimbusml\datasets\_data\train-msltr.sample.csv" />
-    <Content Include="nimbusml\datasets\_data\train-ticketchoice.csv" />
-    <Content Include="nimbusml\datasets\_data\train-twitter.gen-sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\train_fs.csv" />
+    <Content Include="nimbusml\datasets\data\gplv2\airquality.csv" />
```

Comment:
Why was this renamed?

Reply:
Good catch. I must have missed this because there were no conflicts in nimbusml.pyproj. This is also why the CI run fails (`TestPyproj.test_pyproj_existence` and `TestPyproj.test_pyproj_in` - unit tests are passing.